recipes/biophi/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - anarci ==2020.04.23
     - hmmer >=3.1
     - click >=7
-    - sqlalchemy
+    - sqlalchemy <2
```

Comment:
Please add a comment with a reason here. The more info there is in the recipe, the better this can be maintained in the future.